Sources_v3/Database/DTOs/UserDTO.swift

```diff
@@ -102,7 +102,7 @@ extension UserDTO {
 }
 
 extension UserModel {
-    fileprivate static func create(fromDTO dto: UserDTO) -> UserModel {
+    static func create(fromDTO dto: UserDTO) -> UserModel {
```

Comment:
Please use `asModel()` for this and keep this private

Reply:
Fixed in 62b46eb0b0bef360c33b591d1dbceb93446701e2